volume/store/store_unix.go

```diff
@@ -0,0 +1,11 @@
+// +build linux freebsd
+
+package store
+
+import "strings"
+
+// normaliseVolumeName is a platform specific function to normalise the name
+// of a volume. This is a no-op on Unix-like platforms
```

Comment:
That really is a cut and paste error. Removing. Only tolower on Windows. Thanks for catching it!